### DISCUSSION TITLE
blog/dapper-in-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#12] [General]
# blog/dapper-in-aspnet-core/

In this article, we will learn all about Dapper in ASP.NET Core and make a small implementation to understand how it works. Let's not limit it just to Dapper. We will build an application that follows a very simple and clean architecture. In this implementation, we will try to under Repository Pattern and Unit Of Work as well.

https://codewithmukesh.com/blog/dapper-in-aspnet-core/

**Comment by igkprabhu** [comment]
Can you please post example to handle multiple tables in the database.

**Comment by Gabrielpmaricato** [comment]
Thank you for sharing your knowledge. A good way to perform CRUD operations.
